### DISCUSSION TITLE
SQL unable to revert value to NULL

### DISCUSSION THREAD

**Discussion by hardc0l2e** [Budibase/budibase#2276] [Help]
in SQL (PostgreSQL/MySQL) If the row already have an existing value (text, numbers, etc), then wishes to revert it back to NULL, it won't allow you and the system restore it old value back.

![Hho1pOo4dM](https://user-images.githubusercontent.com/1472421/128288459-dfb8b941-7c5f-48fc-918e-ec7e978ad41e.gif)
![7p4uhwVMtE](https://user-images.githubusercontent.com/1472421/128288468-8247dc6b-17bc-49fe-92cf-3a5509740ade.gif)

But when you use the Internal/CSV table it works well, you can easily revert/remove value back to NULL

![dDX9GyiCMA](https://user-images.githubusercontent.com/1472421/128288999-b682a2f6-fdf6-44cb-8e8e-cf72315fb8b5.gif)

**Comment by hardc0l2e** [comment]
Will internal/CSV will have query like features in the future? In that is possible I will drop SQL for now.

**Reply by joebudi** [reply]
Hey!

First of all, I think this might be a bug - I will transfer this to an issue.

Secondly, what do you mean by query-like features? What is it you would like to achieve with internal tables?

**Reply by hardc0l2e** [reply]
Like joining 2 or more tables, for report generation purposes.

**Reply by joebudi** [reply]
Ah. Well, I noticed this was already brought up here #2258 . I'll let the discussion continue there, and convert the SQL / Null issue into a Github issue.
